graphics/vtk/Portfile

```diff
@@ -150,7 +150,7 @@ foreach pyver ${python_versions} {
     set python_branch  "[string index ${pyver} 0].[string range ${pyver} 1 end]"
     set python_major_ver "[string index ${pyver} 0]"
 
-    variant python${pyver} conflicts {*}${other_python_versions} description "Add Python ${python_branch} support" "
+    variant python${pyver} conflicts {*}${other_python_versions} description "Add Python ${python_branch} support" {
```

Comment:
Changing the quotes to braces breaks all but the last defined variant, because the variables like `pyver` that are set in the loop are then evaluated when the variant is executed (after the loop has completed), rather than when it is defined (inside the loop). The result is this:

```
% port deps vtk +python310
Full Name: vtk @9.3.1_1+python310
Build Dependencies:   cmake
Library Dependencies: double-conversion, eigen3, expat, freetype, gl2ps, glew,
                      hdf5, libjpeg-turbo, jsoncpp, libharu, libogg, libtheora,
                      lz4, netcdf-cxx, pugixml, tiff, zlib, python311
```

Reply:
OIC, I was wondering why the quotes were used. But how can quotes be used within the quoted block? Necessary for this new line:
`system -W ${workpath}/build "${prefix}/bin/python${python_branch} setup.py dist_info"`

Reply:
You can backslash escape quotes, and also the `$` on variables that should not be evaluated until later (not sure if there are any of those or not).

Reply:
@Dave-Allured , do you want to include this fix in your update to include python-3.12? Otherwise I can do it after you submit your PR.

Reply:
@jjstickel I would prefer to cancel my python-3.12 update, and have you start a new PR that includes both the 3.12 update and the requested quoting bug fix.  Also remove 3.8 as mentioned in that PR.  I am having a little trouble following these quoting changes.  I also have difficulty with using git correctly.  It is more likely that you will get everything right on the first try.  Okay with you, or should I try harder?   ;-)

Reply:
Yes, that's fine. I see that your changes to add 3.12 and remove 3.8 are trivial. I can start a new PR with that change and the quoting fix.